Add blog post with markdown formatting and proofreading/corrections (#174)
Exodus-Privacy/website: content/post/abortion-bill.fr.md

---

> title: "Conséquences sur la vie privée de la législation sur l'avortement aux États-Unis et dans l'union européenne"

Reviewer: ```suggestion
title: "Conséquences sur la vie privée de la législation sur l'avortement aux États-Unis et dans l'Union Européenne"
```
Par contre le titre peut laisser penser à une législation de chaque côté, mais j'ai pas mieux sous la main.

date: 2022-11-18
drafts: false
---

#### Rappels sur le sujet de l'avortement et la situation

> La Cour suprême des États-Unis a infirmé l'arrêt Roe v. Wade le 24 juin, retirant du niveau fédéral le droit à l'avortement et laissant chaque État des États-Unis libre de décider s'il veut l'autoriser ou le criminaliser. Depuis, 17 États ont interdit ou limité l'avortement. Certains membres du mouvement anti-avortement américain veulent "empêcher les gens de changer d'État pour avorter", relaie le Washington Post[^1]. Ceci est évidemment un problème pour la démocratie américaine, mais ses implications s'étendent bien au-delà.

Reviewer: ```suggestion
La Cour suprême des États-Unis a infirmé l'arrêt Roe v. Wade le 24 juin, retirant du niveau fédéral le droit à l'avortement et laissant chaque État des États-Unis libre de décider s'il veut l'autoriser ou le criminaliser. Depuis, 17 États ont interdit ou limité l'avortement. Certains membres du mouvement anti-avortement américain veulent « empêcher les gens de changer d'État pour avorter », relaie le Washington Post[^1]. Ceci est évidemment un problème pour la démocratie américaine, mais ses implications s'étendent bien au-delà.
```

> Tout d'abord, mais ce n'est pas le but de cet article, ces faits sont un avertissement pour les autres pays puisqu'ils révèlent que même si, la plupart du temps, les démocraties vont dans le sens des droits humains, un retour en arrière est toujours possible, même pour des droits fondamentaux que nous pourrions trop facilement considérer comme acquis. Un premier incident a eu lieu le 30 juin, quand une jeune australienne de Brisbane, durant une escale à Los Angeles, s'est vu demander : "Avez-vous récemment avorté ?" Elle a répondu que non, mais sa réponse semblait suspecte et elle a été placée dans une salle de détention, interrogée à deux reprises, fouillée, photographiée et ses empreintes digitales ont été recueillies. Elle a finalement appris qu'elle ne serait pas autorisée à entrer sur le territoire des États-Unis et qu'elle serait expulsée, forcée à rentrer par le prochain vol pour Brisbane...[^2]

Reviewer: ```suggestion
Tout d'abord, mais ce n'est pas le but de cet article, ces faits sont un avertissement pour les autres pays puisqu'ils révèlent que même si, la plupart du temps, les démocraties vont dans le sens des droits humains, un retour en arrière est toujours possible, même pour des droits fondamentaux que nous pourrions trop facilement considérer comme acquis. Un premier incident a eu lieu le 30 juin, quand une jeune australienne de Brisbane, durant une escale à Los Angeles, s'est vu demander : « Avez-vous récemment avorté ? » Elle a répondu que non, mais sa réponse semblait suspecte et elle a été placée dans une salle de détention, interrogée à deux reprises, fouillée, photographiée et ses empreintes digitales ont été recueillies. Elle a finalement appris qu'elle ne serait pas autorisée à entrer sur le territoire des États-Unis et qu'elle serait expulsée, forcée à rentrer par le prochain vol pour Brisbane…[^2]
```

Apple et Disney ont annoncé leur intention de financer les trajets d'américaines forcées à avorter en-dehors de leur État[^3]. Mais une ébauche rédigée par le Texas Freedom Caucus, un groupe parlementaire de la chambre basse texane, menace d'ores et déjà les cabinets juridiques en évoquant une future législation permettant la condamnation civile et criminelle d'entreprises qui prendraient en charge les frais causés par des avortements hors de leur État[^4].

> Des lois sont passées avant même la décision de la Cour suprême, comme au Texas en Septembre, encourageant les citoyens à traduire en justice des femmes soupçonnées d'avoir avorté ou les personnes qui les auraient aidées - même un taxi ou conducteur Uber qui les aurait supposémment amenées à la clinique, par exemple[^5].

Reviewer: ```suggestion
Des lois sont passées avant même la décision de la Cour suprême, comme au Texas en septembre, encourageant les citoyens à traduire en justice des femmes soupçonnées d'avoir avorté ou les personnes qui les auraient aidées - même un taxi ou conducteur Uber qui les aurait supposémment amenées à la clinique, par exemple[^5].
```
Les mois prennent une majuscule en anglais, pas en français.

La seconde conséquence est liée à la vie privée, et bien entendu nous en parlerons plus en détail dans cet article.

#### Censure et pistage par les réseaux sociaux

> La décision de la Cour suprême a immédiatement été suivie par une énorme quantité de messages sur les réseaux sociaux, où les utilisateurices proposent d'envoyer des pilules abortives aux personnes dont l'accès à l'avortement a été retiré ou est menacé. Mais ces messages ne sont pas tolérés par Facebook et Instagram et sont très rapidement supprimés, parfois quelques secondes après leur publication, menant des utilisateurices à penser que des bots tracent et accèdent au contenu de leurs messages. En réalité, Meta (propriétaire des applications Facebook et Instagram) interdit les messages concernant la vente, le don et le transfert d'armes à feu, de cannabis et... de produits pharmaceutiques (et donc de pilules abortives). Bien sûr, nous pouvons débattre du fait que Meta place au même niveau les médicaments et les armes à feu. Il semble aussi que le système de censure de Meta ne fonctionne pas correctement dès la base : un journaliste de The Verge a proposé des armes et du cannabis dans certains messages, mais contrairement à ses messages proposant des pilules abortives, ils n'ont pas été signalés et supprimés[^6]. Les bots sont-ils plus puissants concernant les pilules abortives ? Une armée de militant·es anti-avortement parcourt-elle les messages, signalant tout ce qu'iels trouvent aux censeurs de Meta ? Ou l'explication se trouve-t-elle ailleurs ? En août, Forbes révèle que Meta a communiqué des messages privés au département de la police du Norfolk, menant au procès d'une jeune fille de 17 ans pour un prétendu avortement. Le Lincoln Journal Star, un journal local du Nebraska, indique qu'un homme de 22 ans a plaidé coupable d'un délit après avoir été accusé d'avoir aidé à enterrer le fœtus de l'adolescente[^7].

Reviewer: ```suggestion
La décision de la Cour suprême a immédiatement été suivie par une énorme quantité de messages sur les réseaux sociaux, où les utilisateurices proposent d'envoyer des pilules abortives aux personnes dont l'accès à l'avortement a été retiré ou est menacé. Mais ces messages ne sont pas tolérés par Facebook et Instagram et sont très rapidement supprimés, parfois quelques secondes après leur publication, menant des utilisateurices à penser que des bots tracent et accèdent au contenu de leurs messages. En réalité, Meta (propriétaire des applications Facebook et Instagram) interdit les messages concernant la vente, le don et le transfert d'armes à feu, de cannabis et… de produits pharmaceutiques (et donc de pilules abortives). Bien sûr, nous pouvons débattre du fait que Meta place au même niveau les médicaments et les armes à feu. Il semble aussi que le système de censure de Meta ne fonctionne pas correctement dès la base : un journaliste de The Verge a proposé des armes et du cannabis dans certains messages, mais contrairement à ses messages proposant des pilules abortives, ils n'ont pas été signalés et supprimés[^6]. Les bots sont-ils plus puissants concernant les pilules abortives ? Une armée de militant·es anti-avortement parcourt-elle les messages, signalant tout ce qu'iels trouvent aux censeurs de Meta ? Ou l'explication se trouve-t-elle ailleurs ? En août, Forbes révèle que Meta a communiqué des messages privés au département de la police du Norfolk, menant au procès d'une jeune fille de 17 ans pour un prétendu avortement. Le Lincoln Journal Star, un journal local du Nebraska, indique qu'un homme de 22 ans a plaidé coupable d'un délit après avoir été accusé d'avoir aidé à enterrer le fœtus de l'adolescente[^7].
```

Tracer me semble hors-sujet ici, l'accès au contenu seul est suffisant, non ?

Author: Changé.

> De plus, les plateformes de réseaux sociaux peuvent filtrer du contenu lié directement ou indirectement à l'avortement dans les États où il est interdit. Des explications, articles, documentaires sur l'avortement peuvent être censurés juste pour respecter la loi... Les services VPN (une connexion sécurisée qui peut servir à masquer l'origine d'une connexion) ont de beaux jours devant eux. Le compte Instagram de Abortion Finder, un site qui permet à des personnes cherchant à avorter de trouver un professionnel, a été brièvement suspendu puis remis en place... Oups[^9].

Reviewer: ```suggestion
De plus, les plateformes de réseaux sociaux peuvent filtrer du contenu lié directement ou indirectement à l'avortement dans les États où il est interdit. Des explications, articles, documentaires sur l'avortement peuvent être censurés juste pour respecter la loi… Les services VPN (une connexion sécurisée qui peut servir à masquer l'origine d'une connexion) ont de beaux jours devant eux. Le compte Instagram de Abortion Finder, un site qui permet à des personnes cherchant à avorter de trouver un professionnel, a été brièvement suspendu puis remis en place… Oups[^9].
```

Pour améliorer leur image de marque, Meta a promis de prendre en charge les coûts pour ses employées devant sortir de leur État pour avorter, et Google a déclaré que ses employées vivant dans des États où l'avortement est interdit peuvent demander à être mutées vers d'autres sites. Malheureusement, déménager sera la seule option simple et sûre qu'ont les femmes concernées et préoccupées par ces sujets pour contourner ces restrictions liberticides.

Si les entreprises de la tech s'inquiètent sincèrement pour la sécurité de leurs utilisateurices, elles doivent arrêter de collecter nos données dès maintenant, et ne plus les avoir quand les mandats arriveront. Au minimum, elles doivent collecter moins de données, les chiffrer, ne pas les partager avec des acteurs douteux, ne pas forcer les utilisateurices à s'authentifier, etc. Bien sûr, cela aura un impact négatif sur leur modèle commercial, mais il est peut-être temps pour elles d'en changer. La société évolue, ces entreprises doivent aussi évoluer si elles prennent en considération leur clientèle.

#### Ordiphones et applications

> Une autre facette de la situation se trouve au niveau des applications pour ordiphones. Si certaines personnes ont deviné la quantité d'informations que les réseaux sociaux ont sur les utilisateurices, peut-être que d'autres ont oublié que les applications sur leurs smartphones collectent énormément de données sensibles. Là encore, certaines applications utilisent des pisteurs Facebook cachés dans des bibliothèques et frameworks gratuits, incluses dans les applications pour gagner du temps de développement. Facebook et Google sont partout. Les ordiphones Android sont à risque, mais c'est aussi le cas pour les appareils Apple même s'ils sont moins exposés.

Reviewer: Sources du « moins exposés » ? Parce qu'à chaque fois que quelqu'un s'y penche c'est en réalité pas spécialement mieux…

Author: Retiré.

> Un type d'application sensible dont on a beaucoup parlé par rapport à la législation sur l'avortement est les applications de suivi de cycle menstruel (55 millions d'utilisateurices actives). Très appréciées par les femmes pour réaliser le suivi de leur santé sexuelle et leur fertilité, leurs données stockées dans le cloud peuvent être partagées avec une liste de partenaires immense et en mouvement constant. Ce partage de données est évidemment une source de revenu pour ces applications, qui peuvent être gratuites au moins en partie. Payer n'est pas une garantie de confidentialité, mais que les utilisateurices gardent en tête que "si c'est gratuit, c'est toi le produit"... Aux États-Unis, les données de santé recueillies par les applications ne sont pas automatiquement protégées par la législation, la loi HIPAA protégeant uniquement les informations recueillies et possédées par les prestataires médicaux. De nombreuses entreprises sont prêtes à payer très cher pour ces données très intimes. Cela pourrait être commercial (vendre des services, des dispositifs médicaux, ou n'importe quelle utilisation innovatrice). Évidemment, comme dit plus haut, la police pourrait également réclamer ces données. Le cycle menstruel peut montrer qu'une femme est enceinte, depuis combien de temps, et peut également révéler qu'elle n'est soudain plus enceinte, potentiellement en violation de la loi de l'État.

Reviewer: ```suggestion
Un type d'application sensible dont on a beaucoup parlé par rapport à la législation sur l'avortement est les applications de suivi de cycle menstruel (55 millions d'utilisateurices actives). Très appréciées par les femmes pour réaliser le suivi de leur santé sexuelle et leur fertilité, leurs données stockées dans le cloud peuvent être partagées avec une liste de partenaires immense et changeante. Ce partage de données est évidemment une source de revenus pour ces applications, qui peuvent être gratuites au moins en partie. Payer n'est pas une garantie de confidentialité, mais que les utilisateurices gardent en tête que « si c'est gratuit, vous êtes le produit »… Aux États-Unis, les données de santé recueillies par les applications ne sont pas automatiquement protégées par la législation, la loi HIPAA protégeant uniquement les informations recueillies et possédées par les prestataires médicaux. De nombreuses entreprises sont prêtes à payer très cher pour ces données très intimes. Cela pourrait être commercial (vendre des services, des dispositifs médicaux, ou n'importe quelle utilisation innovatrice). Évidemment, comme dit plus haut, la police pourrait également réclamer ces données. Le cycle menstruel peut montrer qu'une femme est enceinte, depuis combien de temps, et peut également révéler qu'elle n'est soudain plus enceinte, potentiellement en violation de la loi de l'État.
```

« si c'est gratuit, vous êtes le produit » > Je suis moins confortable avec cette assertion maintenant qui généralise des services commerciaux peu regardants et les place dans le même sac qu'un Wikipédia ou un Linux…

Author: Évidemment, la phrase est un peu gimmicky et il faut faire preuve de nuance et d'analyse. J'ai nuancé en ajoutant "pour de nombreuses entreprises".

> Dans ce contexte, de nombreuses américaines ont commencé à supprimer leurs applications et réclamer la suppression de leur historique. Mais qui peut certifier que les données ont été entièrement effacées et impossibles à restaurer ? Certaines femmes ont commencé à utiliser l'application Clue basée dans l'UE (Berlin), pensant que la RGPD les protègerait. Malheureusement, la RGPD n'est ici pas vraiment une protection à cause de ce que nous avons précisé plus tôt : en premier lieu, même une entreprise européenne doit obénir à une requête légale légitime des autorités des États-Unis, et en second lieu, le stockage d'une application européenne peut ne pas être simple et faire appel à des serveurs situés hors de l'Europe. Là encore, il y a un gros risque que la police d'État exige ces données.

Reviewer: ```suggestion
Dans ce contexte, de nombreuses américaines ont commencé à supprimer leurs applications et réclamer la suppression de leur historique. Mais qui peut certifier que les données ont été entièrement effacées et impossibles à restaurer ? Certaines femmes ont commencé à utiliser l'application Clue basée dans l'UE (Berlin), pensant que le RGPD les protègerait. Malheureusement, le RGPD n'est ici pas vraiment une protection à cause de ce que nous avons précisé plus tôt : en premier lieu, même une entreprise européenne doit obénir à une requête légale légitime des autorités des États-Unis, et en second lieu, le stockage d'une application européenne peut ne pas être simple et faire appel à des serveurs situés hors de l'Europe. Là encore, il y a un gros risque que la police d'État exige ces données.
```
IANAL, mais le RGPD couvre normalement ça - c'est la théorie - en pratique, ça a été démonté par un jugement européen il y a peu parce que le CLOUD Act met tout ça en péril : si un⋅e salarié⋅e américian (même sur sol européen) reçoit une injonction du DoJ de fournir des données et de n'en parler à personne, au vu des risques pour sa personne, il est probable qu'ielle s'exécute.
Par ailleurs je crois que Schrems II estime qu'il y a danger dès qu'un stockage a lieu aux US.

> Certaines start-ups ont vu dans cette situation la possibilité de gagner plus d'argent. L'application Stardust a vu ses installations augmenter de 6 000 % en l'espace d'un week-end, après avoir annoncé qu'elle protègerait les données utilisateur grâce à un chiffrement de bout en bout. La réalité, c'est que l'entreprise n'a toujours pas mis en place ces mesures et a plusieurs lacunes de sécurité (comme partager des numéros de portable avec une entreprise d'analyse tierce)...

Reviewer: ```suggestion
Certaines start-ups ont vu dans cette situation la possibilité de gagner plus d'argent. L'application Stardust a vu ses installations augmenter de 6.000 % en l'espace d'un week-end, après avoir annoncé qu'elle protègerait les données utilisateur grâce à un chiffrement de bout en bout. La réalité, c'est que l'entreprise n'a toujours pas mis en place ces mesures et a plusieurs lacunes de sécurité (comme partager des numéros de portable avec une entreprise d'analyse tierce)…
```

Author: J'ai corrigé les points de suspension mais en français c'est l'espace (insécable) qui est utilisé comme séparateur de milliers.

Lire les politiques de confidentialité de ces applications est laborieux, elles peuvent changer fréquemment, et en fin de compte elles peuvent être déclarées illégales par les autorités gouvernementales.

> Au final, une bonne solution pourrait être d'utiliser des applications qui stockent et traitent les données sensibles localement (au niveau du téléphone) comme Apple Health. Mais là encore, un changement dans le code pourrait mener à une évolution négative au niveau de la vie privée. Des applications entièrement libres pourraient être la seule solution moins risquée. Mais à part Euki et l'application Drip, soutenue par la Fondation Mozilla, toutes les autres stockent les données de santé dans le cloud. Certaines applications ne partagent que des données anonymisées (séparées de toute information pouvant permettre d'identifier l'utilisateurice) avec des tiers, mais des techniques comme l'apprentissage machine pourraient permettre la réidentification de ces données. La plupart des applications partagent des informations pour "être en conformité avec ou répondre aux forces de l'ordre". L'extraterritorialité des lois des États-Unis pourraient également mener des applications non américaines à devoir partager leurs données avec les polices des États. Finalement, de façon évidente, une bonne donnée est une donnée qui n'est ni collectée ni partagée !

Reviewer: ```suggestion
Au final, une bonne solution pourrait être d'utiliser des applications qui stockent et traitent les données sensibles localement (au niveau du téléphone) comme Apple Health. Mais là encore, un changement dans le code pourrait mener à une évolution négative au niveau de la vie privée. Des applications entièrement libres pourraient être la seule solution moins risquée. Mais à part Euki et l'application Drip, soutenue par la Fondation Mozilla, toutes les autres stockent les données de santé dans le cloud. Certaines applications ne partagent que des données anonymisées (séparées de toute information pouvant permettre d'identifier l'utilisateurice) avec des tiers, mais des techniques comme l'apprentissage machine pourraient permettre la réidentification de ces données. La plupart des applications partagent des informations pour « être en conformité avec ou répondre aux forces de l'ordre ». L'extraterritorialité des lois des États-Unis pourrait également mener des applications non américaines à devoir partager leurs données avec les polices des États. Finalement, de façon évidente, une bonne donnée est une donnée qui n'est ni collectée ni partagée !
```

Une solution low-tech utilisant un thermomètre, un calendrier et un crayon reste la plus sûre même si elle est moins séduisante.

> La FTC a intenté un procès à Kochava le 29 août 2022 pour avoir vendu les informations de géolocalisation de "centaines de milliers" de téléphones portables à des groupes anti-avortement. Ces groupes ont utilisé des données géographiques pour identifier des femmes ayant visité des cliniques d'avortement[^10]. Le pire futur est déjà là.

Reviewer: ```suggestion
La FTC a intenté un procès à Kochava le 29 août 2022 pour avoir vendu les informations de géolocalisation de « centaines de milliers » de téléphones portables à des groupes anti-avortement. Ces groupes ont utilisé des données géographiques pour identifier des femmes ayant visité des cliniques d'avortement[^10]. Le pire futur est déjà là.
```

#### Boutiques et e-commerces

Des utilisateurices font remarquer que les boutiques et sites de e-commerce qui stockent l'historique des achats peuvent aussi présenter un risque. L'historique d'achats de produits d'hygiène menstruelle peut aussi être requise par les polices d'État pour révéler des avortements illégaux. Certains proposent de désynchroniser les achats de produits d'hygiène périodique, payer en liquide et annuler les cartes de fidélité (qui identifient les acheteureuses).

#### Recommandations pour les utilisateurices

Cette loi sur l'avortement n'impacte pas que la liberté des femmes, elle peut aussi impacter leur santé [^11][^12]. Il est donc important de donner des conseils pour permettre aux utilisateurices de se protéger.

> Comme d'habitude, nous pouvons conseiller d'utiliser des moteurs de recherche qui recueillent moins de données, comme DuckDuckGo et Startpage, des services de messagerie chiffrés comme Signal ou ProtonMail, des réseaux privés virtuels (VPN) pour cacher votre localisation réelle, de ne pas naviguer en étant identifié⋅e sur des services comme Google, supprimer les cookies à la fermeture du navigateur, configurer votre navigateur pour indiquer aux sites de ne pas vous traquer... Utiliser des solutions logicielles libres ou en local (cloudless) est aussi un bon conseil classique. Évitez d'utiliser les réseaux sociaux pour discuter ou dévoiler des sujets sensibles qui pourraient se retourner contre vous plus tard. N'oubliez pas qu'Internet n'oublie rien, rien n'est réellement privé et quelque chose que vous avez dit il y a dix ans pourrait se révéler un problème dans un futur lointain.

Reviewer: ```suggestion
Comme d'habitude, nous pouvons conseiller d'utiliser des moteurs de recherche qui recueillent moins de données, comme DuckDuckGo et Startpage, des services de messagerie chiffrés comme Signal ou ProtonMail, des réseaux privés virtuels (VPN) pour cacher votre localisation réelle, de ne pas naviguer en étant identifié⋅e sur des services comme Google, supprimer les cookies à la fermeture du navigateur, configurer votre navigateur pour indiquer aux sites de ne pas vous traquer… Utiliser des solutions logicielles libres ou en local (cloudless) est aussi un bon conseil classique. Évitez d'utiliser les réseaux sociaux pour discuter ou dévoiler des sujets sensibles qui pourraient se retourner contre vous plus tard. N'oubliez pas qu'Internet n'oublie rien, rien n'est réellement privé et quelque chose que vous avez dit il y a dix ans pourrait se révéler un problème dans un futur lointain.
```

Je suis pas sûr qu'il soit très avisé de recommander des VPNs sans plus de détails : un mauvais choix pourrait se révéler pire que mieux.

Author: Rajouté "de confiance" pour rappeler de faire preuve d'analyse et se renseigner.

#### Un dernier mot

Une vision globale de la situation nous révèle cruellement la vraie nature du monde dans lequel nous vivons. Nous avons accepté, par facilité, ignorance, confort et appât du gain de laisser un double numérique très détaillé de nous-mêmes.

> Dans une société saine, cela ne devrait pas poser trop de problèmes. Mais d'un autre côté, dans un monde où toutes les dérives et les évolutions sont possibles, l'humain⋅e numérique est à la merci de toutes les forces légales, neutres ou occultes. Il est devenu incroyablement compliqué, voire impossible dans ces conditions, de se protéger contre les abus.

Reviewer: Ya que pour moi que « occulte » sonne vraiment étrange ?

Author: Je vois ça comme "des forces cachées/moins franches" comme du lobbying etc. Ça peut être perçu comme complotiste, mais perso je comprends ce mot différemment et ça ne me pose pas de souci. (Après, my 2 cents : ce n'est pas moi qui ai écrit ça à l'origine :) )

Reviewer: Moi occulte ça me renvoie à de la sorcellerie ou des trucs dans le genre :D

Reviewer: Peut-être faire le même changement côté anglais, qui employait aussi le terme ? (Et mériterait ptetre un pluriel à force, mais je sais pas trop).

L'autre aspect à retenir est aussi que les humain⋅es numériques pourraient assister à une suppression rapide ou progressive de toute vie démocratique. Cette histoire révèle donc la fragilité, parfois recherchée par les autorités, de nos démocraties. Cet électrochoc nous suffira-t-il pour reprendre en main notre destinée numérique ?

> On dit souvent que les cinq étapes du deuil sont le choc et le déni, la colère, la négociation, la dépression et enfin l'acceptation. Nous sommes, à mon avis, entre la colère et la négociation. La dépression nous attend donc, la pénultième étape vers la vraie mise en place de solutions par les citoyen⋅nes.

Reviewer: À qui se réfère « à mon avis » si l'article n'est pas signé ? Attention à la dernière phrase qui sonne un peu prophétique aussi.

Author: Changé "à mon avis" en "nous semblons nous trouver" pour rendre ça plus collectif. J'ai modifié la dernière phrase avec un "espérons-le" pour rendre ça moins prophétique.

> Déjà, des groupes disparates, constitués de personnes qui ne menstruent pas ou plus, développent une technique simple et efficace pour dérouter le pistage : télécharger plusieurs applications, entrer de fausses données, détourner l'utilisation des calendriers périodiques pour dégrader la qualité des données, brouiller le signal, mettre en échec les algorithmes visant à détecter les violations des lois sur l'avortement. C'est une façon de prendre contrôle des outils numériques pour donner une leçon de résistance au règne de la surveillance étatique... L'EFF (Electronic Frontier Foundation) a publié un guide pour aider les femmes : "Ce que les entreprises peuvent faire maintenant pour protéger les droits numériques dans un monde post-Roe". Déjà l'étape 5 ?

Reviewer: ```suggestion
Déjà, des groupes disparates, constitués de personnes qui ne menstruent pas ou plus, développent une technique simple et efficace pour dérouter le pistage : télécharger plusieurs applications, entrer de fausses données, détourner l'utilisation des calendriers périodiques pour dégrader la qualité des données, brouiller le signal, mettre en échec les algorithmes visant à détecter les violations des lois sur l'avortement. C'est une façon de prendre contrôle des outils numériques pour donner une leçon de résistance au règne de la surveillance étatique… L'EFF (Electronic Frontier Foundation) a publié un guide pour aider les femmes : « Ce que les entreprises peuvent faire maintenant pour protéger les droits numériques dans un monde post-Roe ». Déjà l'étape 5 ?
```
